requirements-direct.txt

```diff
@@ -0,0 +1,10 @@
+PyYAML
```

Comment:
Why was this file renamed to requirements-direct.txt? Seems to break with OS convention.